modules/explorer/update.go

```diff
@@ -330,8 +354,8 @@ func dbAddFileContractID(tx *bolt.Tx, id types.FileContractID, txid types.Transa
 	mustPutSet(b, txid)
 }
 func dbRemoveFileContractID(tx *bolt.Tx, id types.FileContractID, txid types.TransactionID) {
-	// TODO: delete bucket when it becomes empty
 	mustDelete(tx.Bucket(bucketFileContractIDs).Bucket(encoding.Marshal(id)), txid)
+	deleteEmptyNestedBucket(tx.Bucket(bucketFileContractIDs), encoding.Marshal(id))
```

Comment:
Yes I guess you are right. That name is bad ^^ Seems strange that there is no built in bucket.IsEmpty function